Nuro/Features/Child/Routine/Views/ChildRoutineView.swift

```diff
@@ -0,0 +1,170 @@
+//
+//  ChildRoutineView.swift
+//  Nuro
+//
+//  Created by Karen Natalia on 05/10/22.
+//
+
+import UIKit
+
+class ChildRoutineView: UIView {
+    
+    private lazy var activityCollectionView: UICollectionView = {
+        let cv = UICollectionView(frame: .zero, collectionViewLayout: collectionViewLayout())
+        cv.backgroundColor = .clear
+        return cv
+    }()
+    
+    private lazy var pageTitle: UILabel = {
+        let label = UILabel()
+        label.text = "Aktivitas XXXX"
+        label.font = .systemFont(ofSize: 64, weight: .bold)
+        label.textAlignment = .center
+        return label
+    }()
+    
+    private lazy var startButton: UIButton = {
+        let button = UIButton(type: .custom)
+        button.configuration = .filled()
+        button.addTarget(self, action: #selector(startActivity), for: .touchUpInside)
+        button.layer.cornerRadius = 17
+        button.clipsToBounds = true
+        
+        let font = UIFont.systemFont(ofSize: 40, weight: .bold)
+        let attributedTitle = NSAttributedString(string: "Mulai", attributes: [NSAttributedString.Key.font: font])
+        button.setAttributedTitle(attributedTitle, for: UIControl.State.normal)
+        
+        return button
+    }()
+    
+    private lazy var stickView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .systemGray
+        view.layer.cornerRadius = 20
+        return view
+    }()
+    
+    var vc: ChildRoutineViewController!
```

Comment:
check how dependency Inversion. I think the view dont need any reference to the VC. you can just pass along the delegate that you need

Reply:
and dont use ! . Try give conditions if the protocol doesnt exists. If you use ! you will opened up some probability for your code to crash some day